files-community/Files
# Feature: Display conflicts modal when pasting in FTP locations

**Resolved / Related Issues**
   Closes #8609 
   Related #12228 (Files would still crash if the root `/` wasn't specified [e.g. `ftp://localhost:21`])

**Changes**
- There was a duplicated class that was causing issues (`FtpManager`), so I left only one
- I created `IFtpStorageService` which is essentially a copy of `IStorageService`. I might remove it and inject the service without its relative interface. @yair100 what are your thoughts on this?

**Limitations**
~Pasting from FTP to FTP isn't showing the conlicts dialog (I'm working on this)~ Fixed

**Validation**
How did you test these changes?
   1. Open app and an FTP connection
   2. Paste an item with a conflicting name
